lib/modules/platform/gerrit/index.md

```diff
@@ -0,0 +1,72 @@
+# Gerrit
+
+## Supported Gerrit Versions
```

Comment:
```suggestion
## Supported Gerrit versions
```

Let's drop the final capitalized word.

Reply:
I've removed the `minimumReleaseAgeLabel` support, it don't make really sense for Gerrit and Renovate.